### ISSUE TITLE
Simconnect does not read AGL, Climb Rate in PMDG 737

### ISSUE THREAD

**Issue by LameLefty** [Fragtality/DynamicLOD#5]
Currently at FL400 in flight - the window recognizes that I am in a flight (session) and is giving me an FPS reading, but does not recognize aircraft flight altitude or climb rate.

![image](https://github.com/Fragtality/DynamicLOD/assets/8063021/50a05a89-0ac1-4d68-a950-0272494561bc)

**Comment by maximls** [comment]
Same here. I tried on Cirrus and TBM. It only sets the initial LOD at 0 AGL and doesn't change LOD with altitude.

**Comment by Fragtality** [comment]
- It was stated that there will be no Support
- Even then, there would be not enough Information to analyze
- Seems like another Application is overwriting the "PLANE ALT ABOVE GROUND" SimVar

**Comment by maximls** [comment]
Thanks for the tip. In my case it was the fsuipc-lvar-modules folder in my Community folder. After renaming it, the SimVars are working now.

I read the 'No Support' after posting my comment :(.

**Comment by Fragtality** [comment]
I'm running FSUIPC with WASM enabled (that fsuipc-lvar-module) all the Time, so it can't be FSUIPC by itself

**Comment by LameLefty** [comment]
I don't recall asking for "Support." I simply posted the issue so you (the developer) can be aware that your application does not appear to be operating correctly with a very popular MSFS add-on in case you wish to investigate and improve it, If you don't care to do so, that's fine. 

EDIT: I dug through the guts of my PC and found an old, obsolete mobiflight folder from who knows what add-on. I deleted it, deleted the DynamicLOD folder from %APPDATA% and reinstalled. I did a quick flight from KLGA to KDCA and everything appears to work now.
